modules/extensions-xml-support/src/test/resources/module-documentation/module-documentation.xml

```diff
@@ -12,10 +12,12 @@
            http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
 
     <property name="aPropertyWithDoc" type="string" doc:description="Documentation for the property"/>
+    <property name="aHiddenPropertyWithDoc" type="string" password="true" doc:description="Documentation for the hidden property"/>
 
     <operation name="operation-with-doc" doc:description="Documentation for the operation">
         <parameters>
             <parameter name="paramWithDoc" type="string" doc:description="Documentation for the parameter" />
+            <parameter name="hiddenParamWithDoc" type="string" password="true" doc:description="Documentation for the hidden parameter" />
```

Comment:
Do you foresee any other type of parameter other than "password"? Like complex parameters, or email parameter or anything like that?. Because in case there may appear other parameters types in the future then it may be better to have different "parameter" elements. Just think about it and let's discuss it offline if necessary.